cli-docs/pow/pow_deals_storage.md

```diff
@@ -0,0 +1,33 @@
+## pow deals storage
+
+List storage deal records
+
+### Synopsis
+
+List storage deal records
+
+```
+pow deals storage [flags]
+```
+
+### Options
+
+```
+      --addrs strings     limit the records to deals initiated from  the specified wallet addresses, treated as and AND operation if --cids is also provided
+  -a, --ascending         sort records ascending, default is sort descending
+      --cids strings      limit the records to deals for the specified data cids, treated as and AND operation if --addrs is also provided
+  -h, --help              help for storage
+  -f, --include-final     include final deals (default true)
```

Comment:
Cool that cobra detects the strange true default and notes it here. (that is automated, the only command doc in our code is `include final deals`)